src/website/views/components/activity.pug

```diff
@@ -0,0 +1,1105 @@
+                      | 3 minute ago
+              .m-activity__end-footer
+                | 04/22/2019
+      .m-activity__day
+        .m-activity__content
+          .m-activity__item
+            .m-activity__start
+              div
+                .a-icon
+                  svg
+                    use(xlink:href='#icon-circle-play-outline')
+                .a-avatar
+                  img(src='../../assets/images/avatar.jpg', alt="avatar")
+            .m-activity__end
+              .m-activity__end-header
+                strong trent.anderson
+                |  deployed
+                strong  Active Directory
+              .m-activity__end-content
+                ul.-stories
+                  li
+                    div
+                      .a-icon.-text--success
+                        svg
+                          use(xlink:href='#icon-check-alt')
+                      | Instance successfully deployed
+                    .-time
+                      | 1 minute ago
+                  li
+                    div
+                      .a-icon.-text--success
+                        svg
+                          use(xlink:href='#icon-check-alt')
+                      | Machine active-directory-vbo8o-1 successfully deployed
+                    .-time
+                      | 2 minute ago
+                  li
+                    div
+                      .a-icon.-text--muted
+                        svg
+                          use(xlink:href='#icon-circle-plus-outline')
+                      | Executing install:Active Directory in machine active-directory-vbo8o-1
+                    .-time
+                      | 2 minute ago
+                  li
+                    div
+                      .a-icon.-text--muted
+                        svg
+                          use(xlink:href='#icon-circle-plus-outline')
+                      | Updating machine IPs
+                    .-time
+                      | 3 minute ago
+              .m-activity__end-footer
+                | 04/21/2019
+  :code(lang='html')
+    <div class="m-activity -feed -stories">
+      <div class="m-activity__day">
+        <div class="m-activity__content">
+          <div class="m-activity__item">
+            <div class="m-activity__start">
+              <div>
+                <div class="a-icon -text--muted">
+                  <svg>
+                    <use xlink:href="#icon-circle-play-outline"></use>
+                  </svg>
+                </div>
+                <div class="a-avatar">
+                  <img src="../../assets/images/avatar.jpg" alt="avatar">
+                </div>
+              </div>
+            </div>
+            <div class="m-activity__end">
+              <div class="m-activity__end-header"><strong>trent.anderson</strong> deployed<strong> Active Directory</strong>
+              </div>
+              <div class="m-activity__end-content">
+                <ul class="-stories">
```

Comment:
@davidojedacl is this needed for any purpose?  If don't, please remove it.

Reply:
done